### PR TITLE
Fixed changes 

### PR DESCRIPTION
**Pull request by cp-shruti-s**
- Reset portfolio section images
- Fixed let's talk button in header section 
- Add globe animation in user review section

### CODE REVIEW

File: vue-frontend/src/components/home-new/PortfolioSection.vue [hunk 1]
```diff
@@ -50,30 +57,34 @@ export default {
...
+          link: "https://nolonely.me/",
         },
         {
           image: togness,
           title: "Togness",
+          aspectRatio: "55%",
           detail:
             "Togness is a photo editor and slideshow maker app for your life's most memorable events like weddings, pets, friends & family, and memorials, etc.",
-          link: "",
+          link: "https://togness.com/",
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/PortfolioSection.vue:87]: Let's add this links to config file incase we will change

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Comment by cp-sumi-k** [comment]
This MR contains fixes for many sections instead of the portfolio section. Is the title of MR seems appropriate?

**Comment by cp-sumi-k** [comment]
Wouldn't a large size image affect the performance of a website?

**Comment by jimmy0251** [comment]
I think we should not add image larger than container of the image.

**Comment by cp-shruti-s** [comment]
Then small size image blur appears @cp-sumi-k

**Comment by cp-sumi-k** [comment]
> Then small size image blur appears @cp-sumi-k

Resize them as per container size

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**